jaeger_client/reporter.py

```diff
@@ -78,9 +77,11 @@ class Reporter(NullReporter):
     def __init__(self, channel, queue_capacity=100, batch_size=10,
                  flush_interval=DEFAULT_FLUSH_INTERVAL, io_loop=None,
                  error_reporter=None, metrics=None, metrics_factory=None,
-                 **kwargs):
+                 sender=None, **kwargs):
```

Comment:
would making sender a `kwargs` be the safest bet? Won't this still run into the issue where sender might be incorrectly interpreted as kwargs?

Reply:
I think `sender=None, **kwargs` is good

Reply:
@black-adder I think that putting `sender` in `kwargs` will hid it more than necessary, without any benefit.
Honestly, if a developer just writes down 8/9 positional arguments, all without naming (aka kwarg), he actually deserves some kind of error :laughing:

Reply:
I agree